src/test/java/com/bitmovin/api/examples/CreateHLSEncodingWithAES128.java

```diff
@@ -0,0 +1,241 @@
+import com.bitmovin.api.encoding.enums.StreamSelectionMode;
+import com.bitmovin.api.encoding.inputs.HttpsInput;
+import com.bitmovin.api.encoding.manifest.hls.HlsManifest;
+import com.bitmovin.api.encoding.manifest.hls.StreamInfo;
+import com.bitmovin.api.encoding.outputs.Output;
+import com.bitmovin.api.encoding.outputs.S3Output;
+import com.bitmovin.api.encoding.status.Task;
+import com.bitmovin.api.enums.Status;
+import com.bitmovin.api.examples.util.H264Representation;
+import com.bitmovin.api.exceptions.BitmovinApiException;
+import com.bitmovin.api.http.RestException;
+import com.mashape.unirest.http.exceptions.UnirestException;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.*;
+
+/**
+ * Created by Raul Vecchione on 26.07.18.
+ */
+public class CreateHLSEncodingWithAES128
+{
+    private static String API_KEY = "<INSERT_YOUR_APIKEY>";
+
+    private static CloudRegion cloudRegion = CloudRegion.AWS_US_WEST_1;
+
+    private static String HTTPS_INPUT_HOST = "<INSERT_YOUR_HTTP_INPUT_HOST>"; // ex.: storage.googleapis.com/
+    private static String HTTPS_INPUT_PATH = "<INSERT_YOUR_PATH_TO_INPUT_FILE>"; // ex.: bitmovin-api-us-west1-ci-input/path/myfile.mp4
+    private static String S3_OUTPUT_ACCESSKEY = "<INSERT_YOUR_ACCESSKEY>";
+    private static String S3_OUTPUT_SECRET_KEY = "<INSERT_YOUR_SECRETKEY>";
+    private static String S3_OUTPUT_BUCKET_NAME = "BUCKET_NAME";
+    private static String OUTPUT_BASE_PATH = "path/to/your/outputs/" + new Date().getTime();
+    private static String AES_STRING_KEY = "<SET_YOUR_STRING_KEY>" ; // 128 bit (16 bytes)  ex.: cab5b529ae28d5cc5e3e7bc3fd4a544d
+    private static String AES_STRING_IV = "<SET_YOUR_STRING_IV>" ; // 128 bit (16 bytes) ex.: 08eecef4b026deec395234d94218273d
+
+    private static BitmovinApi bitmovinApi;
+
+    @Test
+    public void testEncoding() throws IOException, BitmovinApiException, UnirestException, URISyntaxException, RestException, InterruptedException
+    {
+        bitmovinApi = new BitmovinApi(API_KEY);
+        Encoding encoding = new Encoding();
+        encoding.setName("JAVA Encoding HLS With AES-128");
+        encoding.setCloudRegion(cloudRegion);
+        encoding = bitmovinApi.encoding.create(encoding);
+
+        HttpsInput input = new HttpsInput();
+        input.setHost(HTTPS_INPUT_HOST);
+        input = bitmovinApi.input.https.create(input);
+
+        S3Output output = new S3Output();
+        output.setAccessKey(S3_OUTPUT_ACCESSKEY);
+        output.setSecretKey(S3_OUTPUT_SECRET_KEY);
+        output.setBucketName(S3_OUTPUT_BUCKET_NAME);
+        output = bitmovinApi.output.s3.create(output);
+
+        // Encoding Configuration - Adding h264 Representations
+        List<H264Representation> h264Representations = new ArrayList<>();
+        h264Representations.add ( new H264Representation(null, 240, null, 400L, ProfileH264.HIGH));
```

Comment:
Remove the whitespace between `(` and `new`).

Reply:
done